docs/sources/features/datasources/influxdb.md

```diff
@@ -64,21 +64,20 @@ Identifier | Description
 
 {{< docs-imagebox img="/img/docs/v45/influxdb_query_still.png" class="docs-image--no-shadow" animated-gif="/img/docs/v45/influxdb_query.gif" >}}
 
-You find the InfluxDB editor in the metrics tab in Graph or Singlestat panel's edit mode. You enter edit mode by clicking the
-panel title, then edit. The editor allows you to select metrics and tags.
+Find the InfluxDB editor under the metrics tab, while in the edit mode of the Graph or Singlestat panels. 
+Enter edit mode by clicking the panel title, and clicking **Edit**. The editor allows you to select metrics and tags.
```

Comment:
The second sentence of this paragraph should be earlier.

Describe what the editor does, and thus why you might want to get to it.
Then explain how to access the editor.

Reply:
I'm not comfortable making this level of change to the docs.

Reply:
No problem.